doc/api/process.markdown

```diff
@@ -16,6 +16,34 @@ continue.
 as [`process.exit()`][] or uncaught exceptions, and should not be used as an
 alternative to the `'exit'` event unless the intention is to schedule more work.
 
+## Event: 'deprecation'
```

Comment:
Hmmm, if deprecations were still also emitted on `warning` and we moved to deprecate `deprecate` I would be ok with that? Currently it's a little split for not much reason. Maybe that's what has existed outside but I don't see how it will be goof for our users down the line.